manuscript/03.results.md

```diff
@@ -58,13 +58,13 @@ The specific amount of fully bound ligands is dependent on the ligand's propensi
 
 ### Combining Strategies For Superior Selectivity
 
-Each strategy described above provided selectivity benefits in distinct situations, suggesting that they might synergistically improve selectivity when combined. We explored this potential synergy using optimization to determine the ligand specifications which provided optimal selectivity for our theoretical cell populations. More specifically, we optimized the selectivity of a ligand for a particular population, while considering all other populations to be off-target. Our optimization allowed ligand characteristics to vary within biologically plausible bounds, which characteristics we allowed to vary defined our examination of the efficacy of our various ligand engineering strategies. We examined optimizing affinity alone, mixture along with affinity, and valency along with affinity, and finally combined all three strategies (Fig. {@fig:combination}). It should be noted that the parameter space is not convex, and a grid-search strategy was used for strategies where valency and affinity allowed to vary simultaneously in order to locate the global maximum selectivity.
+Each strategy described above provided selectivity benefits in distinct situations, suggesting that they might synergistically improve selectivity when combined. We explored this potential synergy using optimization to determine the ligand specifications which provided optimal selectivity for our theoretical cell populations. More specifically, we optimized the selectivity of a ligand for a particular population, while considering all other populations to be off-target. Our optimization allowed ligand characteristics to vary within biologically plausible bounds, which characteristics we allowed to vary defined our examination of the efficacy of our various ligand engineering strategies. We examined optimizing affinity alone, mixture along with affinity, and valency along with affinity, and finally combined all three strategies (Fig. {@fig:combination}). To examine the efficacy of each approach, we compare the benefits of these strategies to the selectivity demonstrated by an "unoptimized" ligand, a monovalent ligand defined by affinities which were selected to represent biologically plausible initial affinities prior to selectivity engineering. It should be noted that the parameter space is not convex, and a grid-search strategy was used for strategies where valency and affinity allowed to vary simultaneously in order to locate the global maximum selectivity.
```

Comment:
1. Again, I don't think calling it "X0" is the issue, and "unoptimized" seems more illogical. We just need to explain that X0 is the initial point for optimization, and in the method section, justify why we chose the values.
2. Did we use grid-search for this part? I don't remember it. And to be honest, I'm not sure if the space is convex or not. I left a response under that comment, and if it resonates with you we can incorporate that.